include/deal.II/lac/scalapack.h

```diff
@@ -40,57 +40,26 @@ DEAL_II_NAMESPACE_OPEN
+ * are grid dimensions and $Np$ is the total number of processes. The parameters
+ * MB and NB are referred to as row and column block size and determine the
+ * granularity of the block-cyclic distribution.
  *
- * For example, a global real symmetric matrix of size $9\times 9$ is stored in
- * upper storage mode with block sizes 4 × 4:
- * @code
- *                0                       1                2
- *     ┌                                                       ┐
- *     | -6.0  0.0  0.0  0.0  |   0.0 -2.0 -2.0  0.0  |   -2.0 |
- *     |   .  -6.0 -2.0  0.0  |  -2.0 -4.0  0.0 -4.0  |   -2.0 |
- * 0   |   .    .  -6.0 -2.0  |  -2.0  0.0  2.0  0.0  |    6.0 |
- *     |   .    .    .  -6.0  |   2.0  0.0  2.0  0.0  |    2.0 |
- *     | ---------------------|-----------------------|------- |
- *     |   .    .    .    .   |  -8.0 -4.0  0.0 -2.0  |    0.0 |
- *     |   .    .    .    .   |    .  -6.0  0.0 -4.0  |   -6.0 |
- * 1   |   .    .    .    .   |    .    .  -4.0  0.0  |    0.0 |
- *     |   .    .    .    .   |    .    .    .  -4.0  |   -4.0 |
- *     | ---------------------|-----------------------|------- |
- * 2   |   .    .    .    .   |    .    .    .    .   |  -16.0 |
- *     └                                                       ┘
- * @endcode
- * may be distributed using the 2x2 process grid:
- * @code
- *      |   0 2   |   1
- * -----| ------- |-----
- * 0    |   P00   |  P01
- * 2    |         |
- * -----| ------- |-----
- * 1    |   P10   |  P11
- * @endcode
- * with the following local arrays:
- * @code
- * p,q  |             0              |           1
- * -----|----------------------------|----------------------
- *      | -6.0  0.0  0.0  0.0  -2.0  |   0.0 -2.0 -2.0  0.0
- *      |   .  -6.0 -2.0  0.0  -2.0  |  -2.0 -4.0  0.0 -4.0
- *  0   |   .    .  -6.0 -2.0   6.0  |  -2.0  0.0  2.0  0.0
- *      |   .    .    .  -6.0   2.0  |   2.0  0.0  2.0  0.0
- *      |   .    .    .    .  -16.0  |    .    .    .    .
- * -----|----------------------------|----------------------
- *      |   .    .    .    .    0.0  |  -8.0 -4.0  0.0 -2.0
- *      |   .    .    .    .   -6.0  |    .  -6.0  0.0 -4.0
- *  1   |   .    .    .    .    0.0  |    .    .  -4.0  0.0
- *      |   .    .    .    .   -4.0  |    .    .    .  -4.0
- * @endcode
- * Note how processes $(0,0)$ and $(1,0)$ of the process grid store an
- * extra column to represent the last column of the original matrix that
- * did not fit the decomposition into $4\times 4$ sub-blocks.
+ * In the following the block-cyclic distribution of a $10 \times 11$ matrix on
+ * $3\times 3$ Cartesian process grid with the block sizes
+ * $\text{MB}=\text{NB}=2$ is displayed.
  *
- * The choice of the block size is a compromise between a sufficiently large
+ * \htmlonly <style>div.image
+ * img[src="scalapack_block_cycling.png" align="left"]{width:15%;}</style>
+ * \endhtmlonly
+ * @image html scalapack_block_cycling.png "Block-Cyclic Distribution"
```

Comment:
Yes, this looks like the wrong approach. Have you looked at how we size pictures in the `examples/step-*/doc/results.dox` files? I suspect we may be using the `<img>` tag in places but I don't really remember.

Reply:
that's how the size images in `fe_enriched.h`. I pointed @BenBrands to this. I have not looked at other places, but personally I don't think it's wrong.

Reply:
according to the search below there are no other precedents:
```
$ ag "width:" ../include/
../include/deal.II/fe/fe_enriched.h
1025:   * img[src="3source_fe_indices.png"]{width:25%;}</style> \endhtmlonly
```

Reply:
examples show this:
```
<img src="https://www.dealii.org/images/steps/developer/step-57.compare-Re7500.svg" style="width:50%" alt="">
```
but it's entirely different beast not applicable here.

Reply:
Then I don't know any other option.